apps/music/js/ui/views/player_view.js

```diff
@@ -228,19 +230,17 @@ var PlayerView = {
         this.album.classList.remove('hidden-cover-share');
       }
     } else {
-      var titleBar = document.getElementById('title-text');
-
-      titleBar.textContent =
-        metadata.title || navigator.mozL10n.get('unknownTitle');
-      titleBar.dataset.l10nId = metadata.title ? '' : 'unknownTitle';
+      TitleBar.changeTitleText(
```

Comment:
ok, will do. But then I shall patch open.html as well to have the <bdi> element, shouldn't I? I have to admit I don't know how to test the activity...